services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/client/gRPC/GoalStateClientImpl.java

```diff
@@ -80,44 +83,53 @@ public GoalStateClientImpl() {
     private void doSendGoalState(HostGoalState hostGoalState) throws InterruptedException {
```

Comment:
I think can changed the method return value [void] to [ Map<String, List<Goalstateprovisioner.GoalStateOperationReply.GoalStateOperationStatus>> ]，so that return the result to DPM and process it.

Reply:
Good point! Will you change it in your comprehensive version?

Reply:
I have doen it in my version yesterday.

Reply:
> I have doen it in my version yesterday.

Cool. Thanks!